cmd/promtool/resource.go

```diff
@@ -54,37 +54,39 @@ func newResourceSaver(cfg resourceSaverConfig) *resourceSaver {
 		m[filename] = req
 	}
 	return &resourceSaver{
-		writer:            tw,
-		httpClient:        client,
-		fileNameToRequest: m,
-		postProcess:       cfg.postProcess,
+		a,
+		client,
+		m,
```

Comment:
reqs or requests  would be more descriptive.

Reply:
It seems like map variables are named after the value more than the key. It's going to be `map[*http.Request]string`, but I feel using the key (`reqs` instead of `files`) makes more sense in this context!